Framework/Geometry/test/ObjectTest.h

```diff
@@ -392,7 +393,7 @@ class ObjectTest : public CxxTest::TestSuite {
     checkTrackIntercept(track, expectedResults);
   }
 
-  void xtestTrackTwoIsolatedCubes()
+  void testTrackTwoIsolatedCubes()
```

Comment:
Probably the x was just left there by accident.
This test was disabled but it seems to pass both before and after the changes in this branch.